assets/src/blocks/Splittwocolumns/SplittwocolumnsFrontend.js

```diff
@@ -36,6 +36,7 @@ export const SplittwocolumnsFrontend = ({
             <img src={issue_image_src}
                  srcSet={issue_image_srcset}
                  alt={issue_image_title || ''}
+                 title={issue_image_title || ''}
```

Comment:
I don't think that we need `|| ''` here, considering that the backend already has [this check](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/classes/blocks/class-splittwocolumns.php#L131)? 🤔 However in some other blocks it's not there (for instance Gallery), so maybe we need to add this either in PHP or JS to avoid any undefined values

Reply:
Yes make sense, I will remove it from JS part, as its already checked in backend. 👍